src/middlewares/loginMW.ts

```diff
@@ -0,0 +1,9 @@
+import {testData} from '../services/authService';
+
+//doesn't work yet
```

Comment:
Remove comments, comments its a bad practice ...
If we want to tell that: func is not implemented we should throw error with next string "Auth method is not implemented yet"